frontend/src/components/InputAndControlsPanel.tsx

```diff
@@ -18,6 +19,12 @@ import { PromptInput } from "./PromptInput";
 import { TimelinePromptEditor } from "./TimelinePromptEditor";
 import type { TimelinePrompt } from "./PromptTimeline";
 
+// Detect if user is on Windows (Spout is Windows-only)
```

Comment:
I don't think we want to do a frontend check like this because Spout support is a server side concern. If the server is running on Linux and the browser is on Windows (which wouldn't be uncommon if the user uses Runpod) then this check means that the frontend would show Spout even though the server does not support it.

I think this can be avoided if the frontend checks if the server is on Windows perhaps including that information in the hardware info request/response that is used elsewhere.